Makefile

```diff
@@ -0,0 +1,74 @@
+
+# NOTE: This is not the most perfect makefile.
+# There's a few things we could do to improve performance.
+# But since this is new, we can strive for clarity and simplicity here.
+# Could use many of these suggestions:
+# 	https://docs.cloudposse.com/reference/best-practices/make-best-practices/
+
+
+# On _some_ systems, make might not default to bash, which might produce
+# unexpected output/errors. It's assumed that all of the recipes here are written
+# in bash syntax.
+SHELL = /bin/bash
+
+
+default: help
+
+
+# TODO: Add more descriptions to makefile targets
+.PHONY: help
+help:
+	@echo -e "\nThese are some useful commands to work with this project."
+	@echo -e "Please refer to the README.md for more details.\n"
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-20s\033[0m %s\n", $$1, $$2}'
+	@echo -e "\nFor more information, please read the Makefile\n"
+
+
+.env:
+	@echo "Missing '.env' file. Creating one using 'env.example' as a template"
+	@cp env.example .env
+	@echo "Please open the '.env' file and adjust to your local setup!"
+
+
+.PHONY: venv
+venv:  ## Creates a virtualenv at `./.venv`
+	@python -m venv .venv
```

Comment:
Throughout most venv-based projects I've worked on, I've most commonly seen `.venv/` be the name of the venv directory. But I'm cool with either :shrug:

Reply:
I'm all for common practices 👍